### ISSUE TITLE
 when switching from LTE to H+ 

### ISSUE THREAD

**Issue by 392a5237** [kyujin-cho/pixel-volte-patch#120]
Good afternoon.
There is a problem, when switching from LTE to H+ and vice versa,  VoLTE turns off IMS until you turn on airplane mode

**Comment by not-computer-guy** [comment]
Similar issue here. Using Pixel 6. The patch actually works. But every day morning the VoLTE is not working, even though the VoLTE setting is still On in Setting. Have to turn the SIM off and then on again. It seems the SIM on-off method works only after making a call which shows H+.

**Comment by AndorKakas** [comment]
Same issue, every time the phone goes from 4g (LTE) to h (eg. loss of good signal, elevator, parking garage ...) I have to reset the internet for the patch to work.

**Comment by pilipenko179** [comment]
Hello,
The same problem, every time the phone switches from 4g (LTE) to H, H+, or E (for example, loss of a good signal), even when LTE appears, the VoLTE does not work, I have to reset the Internet, turn on the flight mode for the patch to work. VoWIFI working.

**Comment by marsdoe** [comment]
Same issue on SIM1 slot after using this app on my device "Nokia 8.3 5G".
If i using SIM2 - its okay

How to fix SIM1, i have no idea. I cant use erase modemst1 on my device, because bootloader locked

**Comment by fillwithjoy1** [comment]
Hey everyone, it seems like the devices lose connection to 4G (and hence 4G Calling) until you reconnect, which I presume the devices also forget to connect to 4G calling services. Can you try this and see what happens 
1. Call `*#*#4636#*#*`
2. Open `Phone information`
3. In the `Set preferred network type:`, set it to `NR/LTE` (if you have 5G available) or to `LTE`

This forces your phone to only use 4G and disables it from using 3G at all, but see if it improves coverage at all.

**Comment by fillwithjoy1** [comment]
Just pinging again, to see if this is working for anyone.

**Comment by 392a5237** [comment]
Good afternoon. installed LTE. They can't reach me by phone. Since the coverage is not everywhere.

**Comment by fillwithjoy1** [comment]
> Good afternoon. installed LTE. They can't reach me by phone. Since the coverage is not everywhere.

wdym by "installed LTE", did you change a system setting or what?

**Comment by 392a5237** [comment]
Yes, I selected LTE only settings. But I don’t have LTE coverage everywhere in my region.

**Comment by fillwithjoy1** [comment]
> Yes, I selected LTE only settings. But I don’t have LTE coverage everywhere in my region.

So then VoLTE cannot work because it relies on LTE
